flow/project.py

```diff
@@ -2441,7 +2441,7 @@ def print_status(
         :param jobs:
             Only execute operations for the given jobs, or all if the argument is omitted.
         :type jobs:
-            Sequence of instances of :class:`.Job`.
+            Sequence of instances of :class:`~signac.contrib.job.Job`.
```

Comment:
Why do we need the fully qualified name here (and elsewhere that you've changed it)?

Reply:
I think it needs a full path for the intersphinx reference, right? I found some cases where the references weren't rendering correctly, but there may be a shorter way to write this. I decided to make every reference to the signac Job class match the working reference that I found. We can find-and-replace this with a shorter form if you have suggestions.

Reply:
Hmm OK this seems related to the scheduler question. I think there are better ways to disambiguate correctly without requiring fully qualified names, but since there's a string of these PRs right now I'd prefer to just merge these as is and improve linking separately.